@navikt/core/react/src/overlays/action-menu/ActionMenu.tsx

```diff
@@ -952,7 +952,7 @@ const ActionMenuSubContent = forwardRef<
             },
           }}
         >
-          {children}
+          <div className="navds-action-menu__content-inner">{children}</div>
```

Comment:
Looks good on my machine.
Do we still need content-inner?

Reply:
Yes, it makes it so the scrollbar is placed "inside" the content and thus respects border-radius. Without it, the scrollbar will ignore the border-radius 

Before:
<img width="94" alt="Screenshot 2024-10-10 at 14 36 06" src="https://github.com/user-attachments/assets/c73e01e5-75bf-4f47-8968-ecb23d47e732">